lib/Bridge/TolerantParser/SourceCodeFilesystem/ScfClassCompletor.php

```diff
@@ -59,6 +60,10 @@ public function complete(Node $node, string $source, int $offset): Response
 
         $suggestions = [];
         $count = 0;
+        $currentNamespaceDefinition = $node->getNamespaceDefinition();
+        $currentNamespace = (null !== $currentNamespaceDefinition && null !== $currentNamespaceDefinition->name)
+                          ? $currentNamespaceDefinition->name->getText()
+                          : null;
```

Comment:
Extract this to a method `getCurrentNamespace` (in the future might be worth breaking this out into a helper class if required, but a private method would be fine now).